event.go

```diff
@@ -109,7 +118,7 @@ func (e *Event) Run() (string, error) {
 		cmd.Env = append(
 			cmd.Env,
 			"DEADCI_BASEOWNER="+e.BaseOwner,
-			"DEADCI_BASEREPO="+e.BaseRepo,
+			"DEADCI_BASE REPO="+e.BaseRepo,
```

Comment:
We can't change this without breaking backwards compatibility.   It also breaks the convention that the other environmental variables do not seperate the BASE word out.